cloudinit/net/__init__.py

```diff
@@ -1035,9 +1035,16 @@ def _bringup_static_routes(self):
             via_arg = []
             if gateway != "0.0.0.0/0":
                 via_arg = ['via', gateway]
-            util.subp(
-                ['ip', '-4', 'route', 'add', net_address] + via_arg +
-                ['dev', self.interface], capture=True)
+            try:
+                util.subp(
+                    ['ip', '-4', 'route', 'add', net_address] + via_arg +
+                    ['dev', self.interface], capture=True)
```

Comment:
if we set expected return codes to `[0, 2]` could we do without the try / except?

Reply:
Hrm, normally, I'd agree here @igalic, but I think we need the specific handling here for already exists error because we also need to avoid adding the cleanup_cmds, since we didn't setup that route, so I think we should leave that cleanup to whatever set that up in the first place. I'll push a set of changes out here to see what folks think since @rjschwei unassigned himself from the bug. 
 I think we can also shepherd this change in (or something close to it) and revisit higher-level logic to support dracut network config as a separate iteration that avoids us having to use this logic.